gallery/plot_visualization_utils.py

```diff
@@ -0,0 +1,129 @@
+"""
+=======================
+Visualization utilities
+=======================
+
+This example illustrates some of the utilities that torchvision offers for
+visualizing bounding boxes and segmentation masks.
```

Comment:
Maybe refolmulate the line since `make_grid` is added?
(I'm not an expert in English :sweat_smile: )